submissions/Mifaresss/HTML-CSS-Popup/css/nulling-styles.css

```diff
@@ -0,0 +1,9 @@
+/* ========== Nulling styles ========== */

```

Comment:
The most popular name in web development for this file is `reset.css`, also, no need to add this file to this PR because it's not for review.)

Reply:
Ok

Reply:
Is there a need for improvement here?